src/types.rs

```diff
@@ -365,6 +366,7 @@ impl Display for Error {
             Error::MissingDomain => f.write_str("Missing domain while it was explicitly asked to validate it"),
             Error::CouldntLockReader => f.write_str("Couldn't take a lock on the reader mutex. This means that there's already another reader thread is running"),
             Error::Mpsc => f.write_str("Broken IPC communication channel: the other thread probably has exited"),
+            Error::FeeRate(e) => f.write_str(e),
```

Comment:
nit: You can probably leave the error message here, as it becomes a single point of change in the future.